grpc.gemspec

```diff
@@ -289,87 +360,12 @@ Gem::Specification.new do |s|
-  s.files += %w( src/core/lib/security/transport/secure_endpoint.h )
-  s.files += %w( src/core/lib/security/transport/security_connector.h )
-  s.files += %w( src/core/lib/security/transport/security_handshaker.h )
-  s.files += %w( src/core/lib/security/transport/tsi_error.h )
-  s.files += %w( src/core/lib/security/util/json_util.h )
-  s.files += %w( src/core/tsi/fake_transport_security.h )
-  s.files += %w( src/core/tsi/gts_transport_security.h )
-  s.files += %w( src/core/tsi/ssl_transport_security.h )
-  s.files += %w( src/core/tsi/ssl_types.h )
-  s.files += %w( src/core/tsi/transport_security.h )
-  s.files += %w( src/core/tsi/transport_security_adapter.h )
-  s.files += %w( src/core/tsi/transport_security_interface.h )
-  s.files += %w( src/core/ext/transport/chttp2/server/chttp2_server.h )
-  s.files += %w( src/core/ext/filters/client_channel/client_channel.h )
-  s.files += %w( src/core/ext/filters/client_channel/client_channel_factory.h )
-  s.files += %w( src/core/ext/filters/client_channel/connector.h )
-  s.files += %w( src/core/ext/filters/client_channel/http_connect_handshaker.h )
-  s.files += %w( src/core/ext/filters/client_channel/http_proxy.h )
-  s.files += %w( src/core/ext/filters/client_channel/lb_policy.h )
-  s.files += %w( src/core/ext/filters/client_channel/lb_policy_factory.h )
-  s.files += %w( src/core/ext/filters/client_channel/lb_policy_registry.h )
-  s.files += %w( src/core/ext/filters/client_channel/parse_address.h )
-  s.files += %w( src/core/ext/filters/client_channel/proxy_mapper.h )
-  s.files += %w( src/core/ext/filters/client_channel/proxy_mapper_registry.h )
-  s.files += %w( src/core/ext/filters/client_channel/resolver.h )
-  s.files += %w( src/core/ext/filters/client_channel/resolver_factory.h )
-  s.files += %w( src/core/ext/filters/client_channel/resolver_registry.h )
-  s.files += %w( src/core/ext/filters/client_channel/retry_throttle.h )
-  s.files += %w( src/core/ext/filters/client_channel/subchannel.h )
-  s.files += %w( src/core/ext/filters/client_channel/subchannel_index.h )
-  s.files += %w( src/core/ext/filters/client_channel/uri_parser.h )
-  s.files += %w( src/core/ext/filters/deadline/deadline_filter.h )
-  s.files += %w( src/core/ext/transport/chttp2/client/chttp2_connector.h )
-  s.files += %w( src/core/ext/transport/inproc/inproc_transport.h )
   s.files += %w( src/core/ext/filters/client_channel/lb_policy/grpclb/client_load_reporting_filter.h )
   s.files += %w( src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.h )
   s.files += %w( src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb_channel.h )
   s.files += %w( src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb_client_stats.h )
   s.files += %w( src/core/ext/filters/client_channel/lb_policy/grpclb/load_balancer_api.h )
   s.files += %w( src/core/ext/filters/client_channel/lb_policy/grpclb/proto/grpc/lb/v1/load_balancer.pb.h )
-  s.files += %w( third_party/nanopb/pb.h )
```

Comment:
Is the dropping of these headers intended? This seems to be preventing the ruby source-only gem from compiling, since they're referenced but not contained in the package.